subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -4202,13 +4202,15 @@ int lwm2m_socket_start(struct lwm2m_ctx *client_ctx)
 int lwm2m_parse_peerinfo(char *url, struct sockaddr *addr, bool *use_dtls)
 {
 	struct http_parser_url parser;
-#if defined(CONFIG_DNS_RESOLVER)
+#if defined(CONFIG_DNS_RESOLVER) || defined(CONFIG_NET_SOCKETS_OFFLOAD)
 	struct addrinfo hints, *res;
 #endif
 	int ret;
 	u16_t off, len;
 	u8_t tmp;
 
+	LOG_INF("Parse url: %s", log_strdup(url));
```

Comment:
Please use LOG_DBG() here, the info level should be used only when really needed and this does not look like that here.

Reply:
Understood, will do. This print is added only because I need to change to use different LwM2M servers, thus to leave a record in the log.